abz/acousticbrainz.py

```diff
@@ -91,7 +97,14 @@ def submit_features(recordingid, features):
 def process_file(filepath):
     _start_progress(filepath)
     if is_processed(filepath):
-        _update_progress(filepath, ":) done", GREEN)
+        processed_status = get_processed_status(filepath)
+        if processed_status == None:
+	        status = ":) done"
```

Comment:
You're using more than 4-space indentation for this last level, while 4-space indentation levels are used for the rest of the function definition. Also: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
Now the entire function definition follows 4-space indentation :)